### ISSUE TITLE
version 3.5-alpha does not build Cmake generated .vcxproj problem

### ISSUE THREAD

**Issue by EGYPHARAOH** [andlabs/libui#459]
I am using cmake 3.14.0-rc2 gui
it both configure and generate 
BUT when doing
>msbuild all_build.vcxproj
this comes out and it stops
PostBuildEvent:
  Copying libui.res
  setlocal
  D:\CMake\bin\cmake.exe -E copy D:/gitrepo/libui-alpha3.5/build/CMakeFiles/libui.dir/windows/resources.rc.* D:/gitrepo/libui-alpha3.5/build/out/libui.res
  if %errorlevel% neq 0 goto :cmEnd
  :cmEnd
  endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
  :cmErrorLevel
  exit /b %1
  :cmDone
  if %errorlevel% neq 0 goto :VCEnd
  :VCEnd
  Error copying file "D:/gitrepo/libui-alpha3.5/build/CMakeFiles/libui.dir/windows/resources.rc.*" to "D:/gitrepo/libui-alpha3.5/build/out/libui.res".
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: The command "setloc al [D:\gitrepo\libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: D:\CMake\bin\cmake. exe -E copy D:/gitrepo/libui-alpha3.5/build/CMakeFiles/libui.dir/windows/resources.rc.* D:/gitrepo/libui-alpha3.5/build/out/libui.res [D:\gitrepo\libui-alpha3 .5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq  0 goto :cmEnd [D:\gitrepo\libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmEnd [D:\gitrepo\ libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: endlocal & call :cm ErrorLevel %errorlevel% & goto :cmDone [D:\gitrepo\libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmErrorLevel [D:\g itrepo\libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: exit /b %1 [D:\gitr epo\libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :cmDone [D:\gitrepo \libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: if %errorlevel% neq  0 goto :VCEnd [D:\gitrepo\libui-alpha3.5\build\libui.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\Common7\IDE\VC\VCTargets\Microsoft.CppCommon.targets(138,5): error MSB3073: :VCEnd" exited with  code 1. [D:\gitrepo\libui-alpha3.5\build\libui.vcxproj]
Done Building Project "D:\gitrepo\libui-alpha3.5\build\libui.vcxproj" (default targets) -- FAILED.
am I doing something wrong?
what is the solution?
yes I need this older version and I am aware of 4.1-alpha existence

**Comment by andlabs** [comment]
Yes, that alpha has bugs in its cmake files. Use alpha 4 instead.

**Comment by EGYPHARAOH** [comment]
> Use alpha 4 instead

for the sake of ABI i need 3.5
could I copy certain files from Alpha 4 to this so it may work?

**Comment by andlabs** [comment]
No, but you can try editing windows/CMakeLists.txt to change the faulty `copy` command.
